GUI/TimeLog.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.IO;
+using System.Drawing;
+using System.ComponentModel;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace CKAN
+{
+    [JsonObject(MemberSerialization.OptIn)]
+    public class TimeLog
+    {
+        public TimeLog(string dir)
+        {
+            this.dir = dir;
+            this.path = Path.Combine(dir, "playtime.json");
+
+            if (File.Exists(path))
+            {
+                try {
+
+                JObject data = JObject.Parse(File.ReadAllText(path));
+
+                this.InstanceName = (string)data["instance_name"];
+                this.Time = (int)data["time"];
+                }
+                catch {}
+            }
+            else
+            {
+                File.Create(path);
+
+                this.Time = 0;
+            }
+        }
+
+        [JsonProperty("instance_name", NullValueHandling = NullValueHandling.Ignore)]
+        public string InstanceName;
+
+        [JsonProperty("time", NullValueHandling = NullValueHandling.Ignore)]
+        [DefaultValue(0)]
+        public int    Time;
+
+        public string dir;
```

Comment:
Looks like the `dir` member variable isn't used, the `dir` parameter is only used to build `path`. Let's remove this.